add metrics content (#5224)
getsentry/sentry-docs: src/docs/product/sentry-basics/metrics/index.mdx

---
title: Metrics
sidebar_order: 25
description: "Learn more about how Sentry uses your event data to generate metrics."
---

> 

Author: ```suggestion

<Note>

This feature is in beta and is only available if your organization is participating in its limited release. Features in beta are still in-progress and may have bugs. We recognize the irony. If you’re interested in joining this limited release, please reach out at [abc@sentry.io](mailto:abc@sentry.io).

</Note>

```

Because these events represent aggregate data, Sentry doesn't store the full transaction data. So when you're viewing information in [sentry.io](https://sentry.io) that's based on these events, or based on metrics data, you can't drill down into the specifics of these events.

_Indexed_ events are a sub-set of your processed events, and this sub-set is created based on your dynamic sampling settings. These are the events that you decide to keep from your processed events that Sentry stores for you. For these events, we store the complete transaction data, so you can drill down into each event for further insights. In [sentry.io](https://sentry.io), you may see the data generated from these events referred to as _transaction data_ or _events-based_.

![Image of a breakdown of event types](metrics-events-subsets.png)

<!-- TO DO: TALK ABOUT DIFFERENT USE CASES HERE -->

## Custom Performance Measurements

> TBA

Author: ```suggestion
Custom performance metrics on transactions allow you to define metrics that are important to your application and send them to Sentry. Currently, you can set these in the [JavaScript SDK (version `7.0.0` or higher)](/platforms/javascript/performance/instrumentation/performance-metrics/)
```

---
title: Metrics
sidebar_order: 25
description: "Learn more about how Sentry uses your event data to generate metrics."
---

<Note>

> This feature is in beta and is only available if your organization is participating in its limited release. Features in beta are still in-progress and may have bugs. We recognize the irony. If you’re interested in joining this limited release, please reach out at [abc@sentry.io](mailto:abc@sentry.io).

Author: ```suggestion
This feature is in beta and is only available if your organization is participating in its limited release. Features in beta are still in-progress and may have bugs. We recognize the irony. If you’re interested in participating, [join the waitlist](https://sentry.io/for/performance/#updates-signup).
```

Because these events represent aggregate data, Sentry doesn't store the full transaction data. So when you're viewing information in [sentry.io](https://sentry.io) that's based on these events, or based on metrics data, you can't drill down into the specifics of these events.

_Indexed_ events are a sub-set of your processed events, and this sub-set is created based on your dynamic sampling settings. These are the events that you decide to keep from your processed events that Sentry stores for you. For these events, we store the complete transaction data, so you can drill down into each event for further insights. In [sentry.io](https://sentry.io), you may see the data generated from these events referred to as _transaction data_ or _events-based_.

![Image of a breakdown of event types](metrics-events-subsets.png)

<!-- TO DO: TALK ABOUT DIFFERENT USE CASES HERE -->

## Custom Performance Measurements

> Custom performance metrics on transactions allow you to define metrics that are important to your application and send them to Sentry. Currently, you can set these in the [JavaScript SDK (version `7.0.0` or higher)](/platforms/javascript/performance/instrumentation/performance-metrics/)

Author: ```suggestion
Custom performance metrics on transactions allow you to define metrics that are important to your application and send them to Sentry. Currently, you can set these in the [JavaScript SDK (version `7.0.0` or higher)](/platforms/javascript/performance/instrumentation/performance-metrics/).
```
